sdks/python/tox.ini

```diff
@@ -56,27 +56,53 @@ commands =
   python apache_beam/examples/complete/autocomplete_test.py
   python setup.py nosetests --ignore-files '.*py3.py$'
 
+[testenv:py27-pytest]
+commands =
+  python apache_beam/examples/complete/autocomplete_test.py
+  {toxinidir}/scripts/run_pytest.sh {envname} "{posargs}"
+
 [testenv:py35]
 setenv =
   RUN_SKIPPED_PY3_TESTS=0
 commands =
   python apache_beam/examples/complete/autocomplete_test.py
   python setup.py nosetests
 
+[testenv:py35-pytest]
+setenv =
+  RUN_SKIPPED_PY3_TESTS=0
+commands =
+  python apache_beam/examples/complete/autocomplete_test.py
+  {toxinidir}/scripts/run_pytest.sh {envname} "{posargs}"
+
 [testenv:py36]
 setenv =
   RUN_SKIPPED_PY3_TESTS=0
 commands =
   python apache_beam/examples/complete/autocomplete_test.py
   python setup.py nosetests
 
+[testenv:py36-pytest]
+setenv =
```

Comment:
From @robertwb on #9592:
We should probably be using https://tox.readthedocs.io/en/latest/example/basic.html#compressing-dependency-matrix to avoid these kinds of omissions.